server/controllers/userProfileController.js

```diff
@@ -0,0 +1,145 @@
+import db from '../models';
+
+const { Profile } = db;
+/**
+* @description class will implement users profile functionality
+*
+* @class userProfileController
+*/
+class userProfileController {
+  /**
+   * @description - Method to get login user profile
+   *
+   * @param {object} req - Request object
+   * @param {object} res - Response object
+   * @returns {object} - object representing response message
+   * @param {object} next - Return error
+   */
+  static async getLoginUser(req, res, next) {
+    const { userId } = req.body;
+
+    try {
+      const getUser = await Profile.findOne({
+        where: { userId }
+      });
+
+      if (!getUser) {
+        return res.status(404).json({
+          status: 'Fail',
+          message: 'User not found',
+        });
+      }
+
+      const userProfile = {
+        bio: getUser.bio,
+        twiiter: getUser.twitterUrl,
+        facebook: getUser.facebookUrl,
+        role: getUser.role,
+        image: getUser.imageUrl,
+        gender: getUser.gender,
+      };
+
+      return res.status(200).json({
+        status: 'Success',
```

Comment:
Please the status code has already been sent, it should not be sent in the response object. Instead,  set success to true in the response body.